apps/condo/lang/ru/ru.json

```diff
@@ -1136,6 +1136,19 @@
   "AddItemUsingFileUpload": "Через загрузку файла",
   "Status": "Статус",
   "Account": "Аккаунт",
+  "news.component.RecipientCounter.mailing": "Рассылка",
+  "news.component.RecipientCounter.label.properties": "дома",
+  "news.component.RecipientCounter.willReceive.label": "получат",
+  "news.component.RecipientCounter.willReceive.hint": "В {count} помещениях есть \u2028пользователи мобильного \u2028приложения Doma — они получат \u2028новость",
+  "news.component.RecipientCounter.willNotReceive.label": "не получат",
+  "news.component.RecipientCounter.willNotReceive.hint": "В помещениях нет пользователей \u2028мобильного приложения Doma — \u2028они не получат новость. Скачайте \u2028список адресов, чтобы узнать, кто \u2028пользуется приложением.",
+  "news.component.RecipientCounter.toResidentsInAllProperties": "Рассылка по всем адресам",
+  "news.component.RecipientCounter.toResidentsInProperty": "Рассылка по адресу: {address}",
+  "news.component.RecipientCounter.toResidentsInProperties": "Рассылка по адресам: {addressList}{andMore}",
+  "news.component.RecipientCounter.toResidentsInProperties.andMore": " и ещё {count} адреса",
```

Comment:
What about different endings for different meanings?

Reply:
fixed